UKHomeOffice/sgmr-service
# NMSW-44 Update help page

### AC / Description of changes

Update help page linked to from the Footer contains the text changes from the attached word doc
Update help page linked to from the logged in nav contains the text changes from the attached word doc (phone numbers are handled in NMSW-45

### To Test

- Run app locally
- Go to localhost:8080
- DO NOT SIGN IN
- Scroll to the footer
- Click on the help link
- You should see 'For details about submitting your voyage plan and when to do this, you can sign in and information will be available on the section Voyage Plans.
- > Sign in should contain a link
- > The second paragraph should also contain a sign in link
- Click the sign in link
- Sign in to your account
- Click on the help page in the footer again
- > You should see the first paragraph has a link to /voyage-plans
- > The second paragraph should have a link to /pages/help
- > The rest of the content changes should mach those in the word document attached to the ticket
- Click on the help tab in the navigation
- > The text should match the text changes in the document (NOTE: this will not contain changes to the numbers yet)
- > Go to /voyage-plans
- Click on 'Create a new voyage plan' button
- > The information page should not have changed (should not contain the same content as the navigation help as they are separate pages now)



### Notes

* There are two documents attached to the ticket, the correct document is the most recent one and it contains the new contact numbers, even though the numbers will not be updated until NMSW-45

---

**Remember to merge to feature branches not `master`.**
